rand_distr/src/dirichlet.rs

```diff
@@ -93,12 +241,21 @@ where
             }
         }
 
-        Ok(Dirichlet { alpha: alpha.to_vec().into_boxed_slice() })
+        if alpha.iter().all(|x| *x <= NumCast::from(0.1).unwrap()) {
+            // All the values in alpha are less than 0.1.
```

Comment:
This comment is a bit redundant to the code in line 244. Maybe we can instead say how 0.1 was chosen as a cutoff?

Reply:
Done.